airbyte-integrations/connectors/source-freshdesk/source_freshdesk/streams.py

```diff
@@ -34,9 +34,7 @@ def __init__(self, authenticator: AuthBase, config: Mapping[str, Any], *args, **
         # Since this logic rely not on updated tickets, it can break tickets dependant streams - conversations.
         # So updated_since parameter will be always used in tickets streams. And start_date will be used too
         # with default value 30 days look back.
-        self.start_date = (
-            pendulum.parse(config.get("start_date")) if config.get("start_date") else pendulum.now() - pendulum.duration(days=30)
-        )
+        self.start_date = config.get("start_date") or (pendulum.now() - pendulum.duration(days=30)).strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
start_date was a datetime before, now it is a string. 

Is it expected further in the code ?

Reply:
It's expected